src/core/dev_api/include/augru_cell.hpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "openvino/op/op.hpp"
+#include "openvino/op/util/activation_functions.hpp"
+#include "openvino/op/util/rnn_cell_base.hpp"
+
+namespace ov {
+namespace op {
+namespace v0 {
+///
+/// \brief AUGRUSequence operation.
```

Comment:
why not to put such operations to https://github.com/openvinotoolkit/openvino/tree/master/src/common/transformations/include/ngraph_ops ? I suppose we already have such place for internal operations.. 

CC @ilyachur how do you think?

Reply:
done